src/app/parcels/fetchParcelTableData.ts

```diff
@@ -273,9 +273,16 @@ export const getParcelsByIds = async (
         throw new DatabaseError("fetch", "parcel table", logId);
     }
 
-    const congestionCharge = await getCongestionChargeDetailsForParcels(data, supabase);
+    const { data: congestionChargeDetails, error: congestionChargeError } =
+        await getCongestionChargeDetailsForParcelsTable(data);
+
+    if (congestionChargeError) {
+        const logId = await logErrorReturnLogId("Errr");
+        throw new EdgeFunctionError("congestion charge check", logId);
```

Comment:
Although if you're really sick of handling errors, it would be ok for you to make a ticket to cover handling instead of throwing errors for all these functions in `fetchParcelTableData.ts`, as there's quite a lot that needs to be fixed here, and maybe it would be good to do it all at once. Happy for either.

Reply:
Yeah, to be honest I would rather make a ticket and then perhaps tackle it as a separate task - but handling instead of throwing the errors sounds good

Reply:
Should the DatabaseErrors also be handled similarly or are they okay as is?

Reply:
Have made a ticket on Jira

Reply:
> Should the DatabaseErrors also be handled similarly or are they okay as is?

they also need to be handled. I don't think we should be throwing errors anywhere (again, recognise you weren't the one to write these! just explaining)

Reply:
just unresolving so you see this

Reply:
Ok cool, thanks!